css/index.css

```diff
@@ -0,0 +1,70 @@
+/* Media queries */
```

Comment:
Put your css into the "style.css" file instead of having one only for the index page